aspnet/fundamentals/app-state/sample/src/AppState/Model/RequestEntry.cs

```diff
@@ -1,11 +1,7 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
 
 namespace AppState.Model
 {
-    [Serializable]
```

Comment:
@Tratcher can you confirm [Serializable] is required here to allow this to work with Redis/SQL providers of IDistributedCache, please.